config.toml.example

```diff
@@ -4,6 +4,7 @@ api_secret = "golbat"   # Golbat secret required on api calls (blank for none)
 
 stats = false           # Enable creation of pokemon_timing and pokemon_history table (now recommended=false)
 in_memory = false       # Use in-memory storage for pokemon only
+ar_quests = true        # Enable ar_quests scanning by default. Disable to ignore them if your mitm is not sending haveAR flag
```

Comment:
I'm also wondering if this is a true to override, or whether it is what it should be overridden to? Is there a case for supporting setting to have_ar false? A MITM that just clears quests would have a real mixture and I'm not sure this would be workable.
setting to true implies you have a MITM that is AR aware enough not to clear the AR quest?